currency-rates/arch-go.yaml

```diff
@@ -0,0 +1,53 @@
+version: 1
+threshold:
+  compliance: 100
+  coverage: 75
+dependenciesRules:
+  - package: "**.services.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.repo.**"
+        - "**.tools.**"
+  - package: "**.clients.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.tools.**"
+  - package: "**.handlers.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.tools.**"
+  - package: "**.repo.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.tools.**"
+  - package: "**.routers.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.handlers.**"
+        - "**.tools.**"
+  - package: "**.scheduler.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.tools.**"
+  - package: "**.services.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.repo.**"
+        - "**.tools.**"
+  - package: "**.tools.**"
+    shouldOnlyDependsOn:
+      internal:    
+  - package: "**.tests.**"
+    shouldOnlyDependsOn:
+      internal:    
+  - package: "**.cmd.**"
+    shouldOnlyDependsOn:
+      internal:      
+        - "**.app.**"
+        - "**.tools.**"
+
+functionsRules:
+  - package: "**.services.**"
+    maxParameters: 5
```

Comment:
Why did you choose 5?

Reply:
It is my subjective opinion that 5 is the optimal value